ui/desktop/app/controllers/scopes/scope/projects/targets/target.js

```diff
@@ -20,11 +20,23 @@ export default class ScopesScopeProjectsTargetsTargetController extends Controll
 
   queryParams = [{ isConnecting: { type: 'boolean' } }];
 
+  @tracked page = 1;
+  @tracked pageSize = 10;
+  @tracked totalItems = this.model.hosts.length;
+  @tracked paginatedHosts = this.paginateHosts();
+
   @tracked isConnecting = false;
   isConnectionError = false;
 
   // =methods
 
+  paginateHosts() {
```

Comment:
Could this have just been a getter?

Reply:
It sure can. I'll update.

Reply:
asking for learning purposes: when we mean this being a getter, do we mean moving it to the target model? or something else?